### ISSUE TITLE
Move fapolicyd status icons to Gtk3

### ISSUE THREAD

**Issue by tparchambault** [ctc-oss/fapolicy-analyzer#361]
Current implementation uses legacy icons that are on the deprecated list. Also need to add an icon to represent the UNKNOWN state when fapolicyd is not installed or there are comm issues.

**Comment by tparchambault** [comment]
Waiting on issue-123/PR merge prior to creating a new PR. Unfortunately, no straight-forward mappings from "gtk-yes" and "gtk-no" to similarly sized green and red light icons respectively. Completed work on local branch mapping "gtk-yes" to "emblem-default", "gtk-no" to "process-stop", and introducing "edit-delete" to represent the "UNKNOWN" state.
